packages/react/src/components/Dropdown/index.stories.js

```diff
@@ -1,44 +1,100 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { withKnobs } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
 import { Dropdown } from '..';
 import { Item } from '..';
```

Comment:
Would be nice to avoid importing from the `components/index` file as we might end with a circular dependencies
```
import Item from '../Item';
```